docs/requirements.txt

```diff
@@ -2,3 +2,5 @@ sphinx
 sphinx-gallery
 sphinx-toggleprompt
 furo
+git+https://github.com/lab-cosmo/chemiscope@sphinx-gallery
+matplotlib
```

Comment:
I don't think we need matplotlib here. chemiscope is fine, since we are using the corresponding sphinx extension